<launch>

  <node name="pandubot_voice" pkg="pandubot_voice" type="pandubot_voice.py" output="screen">

  </node>

</launch>
